src/System.CommandLine.JackFruit.Tests/TestUtils.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+
+namespace System.CommandLine.JackFruit.Tests
+{
+    internal class TestUtils
```

Comment:
FluentAssertions lets you write methods like these more expressively as extension methods.

Reply:
I haven't written them. Any samples?

Reply:
This is a somewhat complex one but it'll give you the idea: https://github.com/dotnet/command-line-api/blob/master/src/System.CommandLine.Tests/AssertionExtensions.cs

Reply:
Thanks!